src/Admin/BreadcrumbsBuilder.php

```diff
@@ -121,7 +121,7 @@ public function buildBreadcrumbs(
         }
 
         if ('list' === $action) {
-            $menu->setUri(false);
+            $menu->setUri('');
```

Comment:
@franmomu used null in his PR I think

Reply:
Yes, but the version that is used in 3.x accepts strings only